### PR TITLE
Deposit money scenario

### PR DESCRIPTION
**Pull request by anthonyflea**
I've added deposit money scenario.
Please do a code review.

### CODE REVIEW

File: src/main/java/pl/edu/agh/iet/katabank/Account.java [hunk 1]
```diff
@@ -42,12 +42,22 @@ public void setBalance(BigDecimal balance) {
...
+    public void deposit(BigDecimal amount) {
+        checkValidAmount(amount);
+        this.balance = this.balance.add(amount);
+    }
+
+    private void checkValidAmount(BigDecimal amount) {
+        if (amount == null || amount.signum() <= 0) {
+            throw new IllegalArgumentException("Incorrect amount to withdraw: "
+                    + (amount == null ? null : amount.toString()));
+        }
```
Comment (aaalata) [inline src/main/java/pl/edu/agh/iet/katabank/Account.java]: I like the idea of pushing the check to a separate method. Bus as we use it both for withdraw and deposit, the error message could state something like "Incorrect amount: " or "Incorrect amount to process: ", without the "withdraw" word.

Comment (aaalata) [reply src/main/java/pl/edu/agh/iet/katabank/Account.java]: If we change this, we'll need to change that in the groovy tests for withdraw and deposit.

**Review by aaalata — APPROVED** [review]
Other than the error message i think all good.

**Comment by anthonyflea** [comment]
I've corrected error message and tests.
Merging to master.

**Pull request merged**